### DISCUSSION TITLE
Number of workers

### DISCUSSION THREAD

**Discussion by vikashg** [Project-MONAI/MONAI#4118] [General]
Hi all, 
I was wondering if there is a correct way to choose the number of workers in your data loader. Can anyone please shed some light on it. 

Thanks
Vikash

**Comment by Nic-Ma** [comment]
Hi @vikashg ,

I feel this is a question for PyTorch DataLoader.
It depends on your data and HW/SW environment.

Thanks.

**Reply by vikashg** [reply]
Ok I will look into that. 
Thanks
